src/function/algebra/simplify.js

```diff
@@ -1,10 +1,11 @@
-import { isConstantNode, isParenthesisNode } from '../../utils/is'
+import { isParenthesisNode } from '../../utils/is'
 import { factory } from '../../utils/factory'
 import { createUtil } from './simplify/util'
 import { createSimplifyCore } from './simplify/simplifyCore'
 import { createSimplifyConstant } from './simplify/simplifyConstant'
 import { createResolve } from './simplify/resolve'
 import { hasOwnProperty } from '../../utils/object'
+import * as wildcards from './simplify/wildcards'
```

Comment:
Can you replace this wildcard import of wildcards with explicit imports? 😉  

Like `import { isConstantNode, ... } from './simplify/wildcards'`